packages/api/src/parse-cli.js

```diff
@@ -173,6 +173,12 @@ export default function parseCli(argv) {
             "id of the object store that should be used for `record: true` requests that don't otherwise have an os",
           type: 'string',
         },
+        'base-stream-name': {
+          describe:
+            'base stream name to be used in wildcard-based routing scheme.',
+          type: 'string',
+          default: '',
```

Comment:
This is how Go would do it, but I think it would be better to just not have a default and it will be `undefined`.